### ISSUE TITLE
circulating lipid terms

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3121]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1. increased circulating LDL phospholipid level
2. increased circulating VLDL phospholipid level
3. increased circulating HDL triglyceride level
4. increased circulating LDL triglyceride level

>2. Synonyms (e.g., absent spleen)

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1. increase in the concentration in the blood of LDL phospholipid
2. increase in the concentration in the blood of VLDL phospholipid
3. increase in the concentration in the blood of HDL triglycerides
4. increase in the concentration in the blood of LDL triglycerides  
PMID:8301098

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.
>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1. increased circulating non-HDL phospholipid level (MP:0020128)
2. increased circulating non-HDL phospholipid level (MP:0020128)
3. increased circulating triglyceride level (MP:0001552)
4. increased circulating triglyceride level (MP:0001552)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

RGD:sjfl

**Comment by sbello** [comment]
Need to improve definitions by specifying what each complex is
LDL phospholipid - complex of low density lipoproteins, small (18-25 nm) and low density (1.019-1.063 g/ml) particles, and phospholipids, lipids containing phosphoric acid as a mono- or di-ester
  CHEBI:39026
  CHEBI:16247

VLDL phospholipid - complex of very low density lipoproteins, large (30-80 nm) and very low density (0.93-1.006 g/ml) particles, and phospholipids, lipids containing phosphoric acid as a mono- or di-ester
  CHEBI:39027
   CHEBI:16247

HDL phospholipid (fix existing definition) - complex of high density lipoproteins, small (4-13 nm) and dense (greater than 1.063 g/ml) particles, and phospholipids, , lipids containing phosphoric acid as a mono- or di-ester
  CHEBI:39025
   CHEBI:16247

HDL triglyceride - complex of high density lipoproteins, small (4-13 nm) and dense (greater than 1.063 g/ml) particles, and any glyceride resulting from the condensation of all three hydroxy groups of glycerol 
  CHEBI:17855

 LDL triglyceride - complex of high density lipoproteins, small (4-13 nm) and dense (greater than 1.063 g/ml) particles, and any glyceride resulting from the condensation of all three hydroxy groups of glycerol 
  CHEBI:17855

What does "non-HDL phospholipid mean"?  MGI has 1 annotation from PMID:23846500 for the non-HDL terms. From this it seems that non-HDL is just the fraction that is not HDL and would cover LDL and VLDL along with free. So using this a parent makes sense. But, the definition needs to be improved.

**Comment by sbello** [comment]
Added abnormal, increased and decreased circulating levels
